packages/bisonapp/commands/dbCommands/migrate.ts

```diff
@@ -0,0 +1,31 @@
+import yargslib from "yargs";
+import { runCLICommands } from "../../lib/helpers";
+import { handler as generatePrismaClient } from "../buildCommands/prisma";
+
+export const command = "migrate";
+export const description = "Migrate the database";
+
+export const builder = (yargs: yargslib.Argv<{}>) => {
+  yargs.option("dbClient", {
+    default: true,
+    description: "Generate the Prisma client",
+    type: "boolean",
+  });
+};
+
+export const handler = async ({ dbClient = true }) => {
+  let args = ["migrate up", "--experimental"];
```

Comment:
We will need to update this with the new prisma 2.19.0 commands

Reply:
This prisma is way out of date too. They are up to 3.1.1 https://github.com/prisma/prisma/releases/tag/3.1.1